node-hub/dora-gr00t/dora_gr00t/main.py

```diff
@@ -0,0 +1,328 @@
+"""TODO: Add docstring."""
+
+import json
+import logging
+import os
+import sys
+from typing import Any, Dict
+
+import numpy as np
+import pyarrow as pa
+import torch
+from dora import Node
+
+# Configure logging
+logging.basicConfig(
+    level=logging.INFO,
+    format='%(asctime)s - %(name)s - %(levelname)s - %(message)s'
+)
+logger = logging.getLogger("dora_gr00t")
+
+# Ensure the Isaac-GR00T submodule is in the Python path
+isaac_gr00t_path = os.path.join(os.path.dirname(os.path.dirname(__file__)), "isaac_gr00t")
+if os.path.exists(isaac_gr00t_path):
+    sys.path.append(isaac_gr00t_path)
+    logger.info(f"Added Isaac-GR00T submodule to Python path: {isaac_gr00t_path}")
+else:
+    logger.warning(f"Isaac-GR00T submodule not found at {isaac_gr00t_path}")
+
+try:
+    # Try to import from the submodule or installed package
+    from gr00t.experiment.data_config import DATA_CONFIG_MAP
```

Comment:
I don't think that an absolute import would work here.

```suggestion
    from .groot.experiment.data_config import DATA_CONFIG_MAP
```